python/cloud-run-django-hello-world/requirements.txt

```diff
@@ -1,2 +1,3 @@
 django==3.0.7
-requests==2.23.0
\ No newline at end of file
+requests==2.23.0
+ptvsd==4.3.2
```

Comment:
This feels a bit weird to me.
I understand that we need to package this with the code, but this is for debugging purposes rather than one of the app dependencies. Is there another way to bundle it or at the very least, maybe add a comment to explain that this is not part of the "core" app?

Reply:
Sounds good. I don't know another way to bundle `ptvsd` that will be picked up by buildpacks, but I'm also not very experienced with Python.

How does a comment like `This is required for debugging.` sound?

Reply:
Updated with comments